nx-cloud.env

```diff
@@ -0,0 +1 @@
+NX_CLOUD_ACCESS_TOKEN=
```

Comment:
@ali-hosseini-fs How does this token work? Will we have a single token for everyone or each dev needs to create its own 🤔

Reply:
just one token for whole project
so everyone can have access to nx cloud system to use each others cached materials

Reply:
as far as i know it just one token for whole project
so everyone can have access to nx cloud system to use each others cached materials